app/site/questions.js

```diff
@@ -37,6 +37,18 @@ export const questionsParams = {
     component: "residents-with-disabilities",
     description: "Burnside is the neighborhood with the highest disability rate. What percentage of residents in Burnside, Chicago have an included disability?",
   },
+  "most-common-dropoff": {
```

Comment:
Can you add taxi to these to distinguish from other modes of transit? Or are you planning to show multiple modes of transit?

Reply:
Oh right I'll add taxi to them!